Core.DomainServices/Queries/SystemUsage/QueryReadModelsWhichShouldBecomeActive.cs

```diff
@@ -29,7 +29,27 @@ public IQueryable<ItSystemUsageOverviewReadModel> Apply(IQueryable<ItSystemUsage
                             // Include only if not expired or no expiration defined
                             (x.SourceEntity.ExpirationDate == null || currentTime <= x.SourceEntity.ExpirationDate)
                         )
+                    ) &&
```

Comment:
we want the object to be recalculated if any of the conditions calls for recalculation.... that means logical OR right? between the groups of validations